Cargo.toml

```diff
@@ -6,3 +6,7 @@ members = [
     "frost-ristretto255",
     "frost-p256",
 ]
+
+# This is a development helper binary.
+# To run it, execute `cargo run --manifest-path gendoc/Cargo.toml`
+exclude = ["gendoc"]
```

Comment:
```suggestion
    "frost-p256",
    "gendoc"
]
```

Reply:
I won't just apply this as it makes `gendoc` the default executable when `cargo run` is done in the root of the workspace, which I'm not sure we want?

Reply:
I'll try tinkering with the manifest to make it work

Reply:
If we use `default-members` then `cargo run` won't run it, but you will need to use `cargo check --workspace` to check everything. What do you think?

Reply:
Resolved for now with https://github.com/ZcashFoundation/frost/pull/130/commits/07ebaa3a1608a7acb96209f2ac80f06e264b6af1